templates/storyboards/swift3.stencil

```diff
@@ -24,89 +20,74 @@ protocol StoryboardSceneType {
 }
 
 extension StoryboardSceneType {
-  static func storyboard() -> UIStoryboard {
-    return UIStoryboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
-  }
-
-  static func initialViewController() -> UIViewController {
-    guard let vc = storyboard().instantiateInitialViewController() else {
-      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
-    }
-    return vc
+  static func storyboard() -> {{prefix}}Storyboard {
+    return {{prefix}}Storyboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
   }
 }
 
 extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
-  func viewController() -> UIViewController {
```

Comment:
@AliSoftware thoughts on that last comment (about removing the cases and using `static var`s)? It would also mean removing these functions, as we'd directly call `.storyboard().instantiateViewController(withIdentifier: ...)` and cast if needed.